.dockerignore

```diff
@@ -1,6 +1,9 @@
 ### IDE files
 .idea/
 
+### Composer files
+auth.json
```

Comment:
nope. This does not break anything. In install.yml and in dev usage we bind mount the project directly to the container. So we still got access to auth.json there.